canonical-web-and-design/ubuntu.com
# Move /smartstart to /core/smartstart

## Done

- Move /smartstart to /core/smartstart
- Added it as 'Consulting' to the Core nav
- Add redirects
- Update links in site
- Moved core in the footer to allow for new pages.
 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/core/smartstart
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the site works as expected


## Issue / Card

Fixes #3706
